src/Spout/Writer/Style/Style.php

```diff
@@ -56,11 +56,27 @@ class Style
     /** @var bool Whether specific font properties should be applied */
     protected $shouldApplyFont = false;
 
+    protected $verticalAlignment = 'center';
```

Comment:
There are 2 different changes here: alignment and number format. Can you please split these changes into 2 branches? It will reduce the scope of each commit and make the review easier/faster. Thanks!